eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -348,6 +359,10 @@ jobs:
           extraBuildProperties="$extraBuildProperties /p:TargetArchitecture=${{ parameters.targetArchitecture }}"
         fi
 
+        if [[ -n "${{ parameters.buildPass }}" ]]; then
+          extraBuildProperties="$extraBuildProperties /p:DotNetBuildPass=${{ parameters.buildPass }}"
```

Comment:
Does this also need to be passed into the [windows build.cmd](https://github.com/dotnet/sdk/pull/43193/files#diff-dc549eeb99aa73960e74c9d4c160a3c76e3e414398bb348c1d5365aac0369ad5R261)?

Reply:
Oh no, now I need to remember the COMMAND.COM conditional syntax